docs/cypress-testing-library/intro.md

```diff
@@ -52,8 +52,6 @@ You can find
 ## Examples
 
 To show some simple examples (from
-[cypress/integration/query.spec.js](https://github.com/testing-library/cypress-testing-library/blob/master/cypress/integration/query.spec.js)
-or
 [cypress/integration/find.spec.js](https://github.com/testing-library/cypress-testing-library/blob/master/cypress/integration/find.spec.js)):
```

Comment:
```suggestion
[cypress/integration/find.spec.js](https://github.com/testing-library/cypress-testing-library/blob/97939da7d4707a71049884c0324c0eda56e26fc2/cypress/integration/find.spec.js)):
```

So that we have a permalink. Otherwise this can break again since this document and the JS file live very far apart.

Reply:
Makes sense! Thank you.